### DISCUSSION TITLE
ueli

### DISCUSSION THREAD

**Discussion by lighttigerXIV** [catppuccin/catppuccin#2056] [Port Requests]
### Will you make the port?

- [X] Yes, I'm willing to make the port

### How can the suggested port be themed?

I had already [did everything](https://github.com/lighttigerXIV/ueli). I just need to update the image links when the repo is moved :pray:

### Submission Guidelines

- [X] I have read and followed Catppuccin's submission guidelines.

**Comment by sgoudham** [comment]
Closed due to #2060
